src/PageNavigation/NextPageLink.tsx

```diff
@@ -0,0 +1,19 @@
+import React from '@portal/react';
+import type { ResolvedNavLinkItem } from '@portal/types';
+import { usePageData } from '@portal/hooks/usePageData';
+import { Button } from '@theme/ui';
+
+type NextPageLink = ResolvedNavLinkItem | null;
+
+export function NextPageLink() {
+  const { nextPage }: NextPageLink = usePageData('nextPage');
```

Comment:
let's just have `usePageData()`. The one with argument is harder to type with ts and harder to document.  Also it doesn't provide any perf benefits because we anyways have all the data in the context.

Reply:
Agree, because anyway we're making destructuring